src/components/Organisms/TodoCard/index.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import COLOR from "../../../variables/color";
+import { AddTaskButton } from "../../Atoms/AddTaskButton/index";
+import { Task } from "../../Molecules/task/index";
+
+export const TodoCard = () => {
+  const [taskList, setTaskList] = useState([]);
+
+  const onAddTaskButtonClick = () => {
+    const newTask = {
+      name: "",
+      initializing: true,
+    };
+    setTaskList([...taskList, newTask]);
+  };
+
+  const onTaskComplete = (index) => {
+    let newTaskList = taskList.filter((value, Index) => Index !== index);
```

Comment:
```suggestion
    const newTaskList = taskList.filter((value, Index) => Index !== index);
```

再代入することがないのであれば、const を使うのが望ましいです！